test/Microsoft.AspNetCore.Mvc.Abstractions.Test/ModelBinding/ModelStateDictionaryTest.cs

```diff
@@ -12,20 +14,125 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
     public class ModelStateDictionaryTest
```

Comment:
Though the enumerators are a big chunk of code, doesn't look like they're tested at all. Also lacking tests for `ModelStateEntry.GetModelStateForProperty()`.